### PR TITLE
PLANET-7512 Update Take Action Boxout styles

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-7512](https://jira.greenpeace.org/browse/PLANET-7512)

**Requirements:**

- [x] The layout is divided evenly between the image and the content, each taking up 50% of the block's width.
- [x] Stretch the button width to the full width of the content.
- [x] On mobile, the layout is stacked vertically.
- [x] The sticky version of the block will remain as is.
- [x] Allow the hover state of the CTA button only when the user hovers over it.
- [x] Add a placeholder when no image is selected.

**Related PR:** https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1234

### Testing

- [Sticky TAB](https://www-dev.greenpeace.org/test-pluto/story/2485/post-with-sticky-tab-block/)
- [Non-sticky TAB](https://www-dev.greenpeace.org/test-pluto/story/2487/post-with-non-sticky-tab-block/)
- [Sticky TAB with no image](https://www-dev.greenpeace.org/test-pluto/story/2867/sticky-tab-with-no-image/)
- [Non-sticky TAB with no image](https://www-dev.greenpeace.org/test-pluto/story/2865/tab-with-no-image/)

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [pluto](https://www-dev.greenpeace.org/test-pluto) | [admin](https://www-dev.greenpeace.org/test-pluto/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-pluto/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-pluto) | [composer-local.json](https://github.com/greenpeace/planet4-test-pluto/blob/main/composer-local.json)

:watch: 2024.09.03 10:37:59

**Review by mardelnet — COMMENTED** [review]
Good job, @mleray 
I have some comments for you:

1. Styles in the editor look very different when a "custom link" is chosen instead of an action page:
Screenshot from 2024-09-04 09-16-06

2. What should happen when no image is selected? Should the text and button go full-block width? Should we use a default image?
Screenshot from 2024-09-04 09-18-06

3. When the mouse pointer hovers over the block, the entire element is clickable and the title shows its hover state. Is it that expected?

4. The sticky block does not show the description. Is that expected?

**Comment by mleray** [comment]
> 1. Styles in the editor look very different when a "custom link" is chosen instead of an action page:

I'll have a look 👀 

> 2. What should happen when no image is selected? Should the text and button go full-block width? Should we use a default image?

I'll ask Houssam about this, I don't know! 

> 3. When the mouse pointer hovers over the block, the entire element is clickable and the title shows its hover state. Is it that expected?

Yes, that is expected, I've discussed it with Houssam on Slack:

Screenshot 2024-09-04 at 10 15 23

> 4. The sticky block does not show the description. Is that expected?

Yes, that follows the [designs](https://www.figma.com/file/9NtbY8n3at8uOEJTsLrETb/P4-Design-System?type=design&node-id=1269%3A52764&mode=design&t=YH2F6sOpuuU9uwkf-1)!

File: assets/src/blocks/TakeActionBoxout/TakeActionBoxoutFrontend.js [hunk 1]
```diff
@@ -38,18 +38,18 @@ export const TakeActionBoxoutFrontend = ({
       {excerpt &&
         <p className="boxout-excerpt" dangerouslySetInnerHTML={{__html: excerpt}} />
       }
+      {link && linkText &&
```
[the remark this answers is not in the document]
Comment (mleray) [reply assets/src/blocks/TakeActionBoxout/TakeActionBoxoutFrontend.js:41]: It was moved within the `boxout-content` div

File: templates/blocks/take-action-boxout.twig [hunk 2]
```diff
@@ -35,19 +35,19 @@
...
+                        data-ga-action="Call to Action"
+                        data-ga-label="n/a"
+                        href="{{ boxout.link|default('#') }}"
+                        {{ boxout.new_tab and boxout.link ? 'target="_blank"' }}
+                    >
+                        {{ boxout.link_text }}
+                    </a>
+                {% endif %}
 			</div>
-			{% if ( boxout.link and boxout.link_text ) %}
```
[the remark this answers is not in the document]
Comment (mleray) [reply templates/blocks/take-action-boxout.twig:39]: It was moved within the `boxout-content` div

File: assets/src/scss/blocks/TakeActionBoxout/TakeActionBoxoutStyle.scss [hunk 3]
```diff
@@ -70,138 +91,122 @@
 
   .btn {
     _-- {
-      font-size: 12px;
+      font-size: 15px;
```
Comment (mardelnet) [inline assets/src/scss/blocks/TakeActionBoxout/TakeActionBoxoutStyle.scss:94]: Should we use a variable instead of `px`?

**Review by mardelnet — COMMENTED** [review]
Good job, @mleray 
I noticed in the admin editor that blocks using custom links lose the button background color. Also (and this is very picky-nicky, and so optional) in the editor, the mouse cursor does not change to "pointer" on block hover.

Screenshot from 2024-09-05 09-51-22

**Review by mardelnet — APPROVED** [review]
Good job!

**Pull request merged**